internal/pkg/apikey/apikey_integration_test.go

```diff
@@ -22,7 +22,7 @@ const testFleetRoles = `
 	"fleet-apikey-access": {
 		"cluster": [],
 		"applications": [{
-			"application": ".fleet",
+			"application": "fleet",
```

Comment:
Why do we have a copy here instead of using the one from the actual implementation? Using a copy means we could have a passing test but the actual implementation does not work.